cli/src/commands/build_toolchain.rs

```diff
@@ -112,32 +116,67 @@ impl BuildToolchainCmd {
     })?;
 
     // Apply patches
-    Command::new("patch")
+    // We allow this to fail, but want to warn the user if it did.
+    let patch_output = Command::new("patch")
       .args([
         "-f",
+        "-N",
         "-p1",
         "-i",
         toolchain_dir.join("patches/rust.patch").to_str().unwrap(),
       ])
       .current_dir(&rust_dir)
+      .output()
+      .expect("Failed to run patch command");
```

Comment:
Why panic here? In other places the error is bubbled up with `?`. How about:
```suggestion
      .output()?;
```